forage-android/src/main/java/com/joinforage/forage/android/LDManager.kt

```diff
@@ -49,7 +49,7 @@ internal object LDManager {
     internal fun initialize(app: Application, ldConfig: LDConfig) {
         val contextKind = ContextKind.of(LDContextKind.SERVICE)
         val context = LDContext.create(contextKind, LDContexts.ANDROID_CONTEXT)
-        client = LDClient.init(app, ldConfig, context, 0)
+        client = LDClient.init(app, ldConfig, context, 1)
```

Comment:
is this the LD fix? Was this merged already?

Reply:
Yup this is the fix and it's going in!